extensions/visualizations/visualizations.js

```diff
@@ -66,10 +66,25 @@ oppia.directive('oppiaVisualizationFrequencyTable', [function() {
     scope: {},
     templateUrl: 'visualizations/FrequencyTable',
     controller: [
-      '$scope', '$attrs', 'HtmlEscaperService',
-      function($scope, $attrs, HtmlEscaperService) {
+      '$scope', '$attrs', 'HtmlEscaperService', 'SolutionVerificationService',
+      function(
+          $scope, $attrs, HtmlEscaperService, SolutionVerificationService) {
         $scope.data = HtmlEscaperService.escapedJsonToObj($attrs.data);
         $scope.options = HtmlEscaperService.escapedJsonToObj($attrs.options);
+        $scope.isAddressed = $scope.data.map(function() { return false; });
+        $scope.addressedCellContent = function(i) {
+          return $scope.isAddressed[i] ? '<span>Yes</span>' : '<span>No</span>';
```

Comment:
Don't do any HTML stuff in the JS; keep all HTML formatting to the HTML. (See previous comment about using ng-if.)

Reply:
Done.